solr_conf_4_testing/schema.xml

```diff
@@ -235,8 +235,6 @@
     <dynamicField name="*_facet" type="string" stored="true" indexed="true" multiValued="true" />
     <dynamicField name="*_display" type="string" stored="true" indexed="false" multiValued="true"/>
 -->
-    <dynamicField name="*_si" type="string" stored="true" indexed="true" omitNorms="true" />
```

Comment:
I did a pretty decent check.  No such SearchWorks fields.  No spotlight-dor-resources or other spotlight/ exhibits gem code.  Checked an old exhibits Solr on sul-solr and feigenbaum.